scripts/pipeline_generator/pipeline_generator_helper.py

```diff
@@ -0,0 +1,26 @@
+from typing import List
+
+def get_build_commands(container_registry: str, buildkite_commit: str, container_image: str) -> List[str]:
```

Comment:
any unit test? or is this so simple that does not require one?

maybe we should have some form of integration test to test these commands.

Reply:
It's just replacing placeholders for container registry, container image, and commit so I don't think unit test is needed here.